homeassistant/components/fitbit/sensor.py

```diff
@@ -503,10 +504,20 @@ class FitbitSensorEntityDescription(SensorEntityDescription):
 
 FITBIT_RESOURCE_BATTERY = FitbitSensorEntityDescription(
     key="devices/battery",
-    name="Battery",
+    translation_key="battery",
     icon="mdi:battery",
     scope=FitbitScope.DEVICE,
     entity_category=EntityCategory.DIAGNOSTIC,
+    has_entity_name=True,
+)
+FITBIT_RESOURCE_BATTERY_LEVEL = FitbitSensorEntityDescription(
+    key="devices/battery_level",
+    translation_key="battery_level",
```

Comment:
This sensor can use device class naming, although the old sensor uses that name now, what do you think about switching that around?

Reply:
That sounds better -- though does that rename the entity? I imagine this may be a breaking change depending on when people upgraded to get unique ids (available in this next same release).

Reply:
The entity would be called _battery. So I think it does, but I'm not sure what the impact is

Reply:
OK, went ahead and swapped the names.

Reply:
I was mistakenly thinking the unique ids were new, but they are not new. Reverting this based on Martin's comment.